vms/platformvm/vm.go

```diff
@@ -189,13 +202,25 @@ func (vm *VM) Initialize(
 		txExecutorBackend,
 		validatorManager,
 	)
-	vm.Network = network.New(
+
+	vm.Network, err = network.New(
 		txExecutorBackend.Ctx,
 		vm.manager,
 		mempool,
 		txExecutorBackend.Config.PartialSyncPrimaryNetwork,
 		appSender,
+		txGossipThrottlingPeriod,
+		txGossipThrottlingLimit,
+		txGossipBloomMaxItems,
+		txGossipBloomFalsePositiveRate,
+		txGossipBloomMaxFalsePositiveRate,
+		registerer,
 	)
+	if err != nil {
+		return fmt.Errorf("failed to initialize network: %w", err)
+	}
+	go vm.Network.Gossip(ctx)
```

Comment:
q: I wonder if we should start gossiping only after vm.Builder is instantinated?